.sphinx/_templates/base.html

```diff
@@ -0,0 +1,7 @@
+{% extends "furo/base.html" %}
+
+{% block theme_scripts %}
+<script>
+  const github_url = "{{ github_url }}";
```

Comment:
I'm not sure if this is supposed to be picking the url up automatically from somewhere, but it doesn't seem to be working when I make the site locally. The feedback button disappeared completely. Is there somewhere specific that the url has to be added, because I can't find the place in conf.py where it should be done. We'll have to take a look when you get back :)

Reply:
It takes the URL from here: https://github.com/canonical/sphinx-docs-starter-pack/blob/c1d7aa0cd37d501f5ac1ce0622371e0016734902/conf.py#L26
Looks like this change isn't in the conf.py on your branch yet, so you need to rebase your branch to the latest master - then it should work. :)